src/main/resources/templates/map-location.html

```diff
@@ -10,7 +10,7 @@
 <body>
 
 <div class="govuk-width-container" layout:fragment="content">
-    <a href="/dashboard" class="govuk-back-link">Back</a>
+    <a href="/journeys" class="govuk-back-link">Back</a>
```

Comment:
This template serves as both update and create so this link may need to smarter in terms of where it goes...